website/source/guides/identity/secure-intro.html.md

```diff
@@ -133,6 +137,37 @@ this model can be applied regardless of where the applications are running.
   - [Cubbyhole Response Wrapping](/guides/secret-mgmt/cubbyhole.html)
 
 
+### Vault Agent
+
+Vault agent allows the implementation of the [platform integration](#platform-integration)
+pattern easier and faster.
+
+For those [supported auth methods](/docs/agent/autoauth/methods/index.html),
+Vault agent can automate the authentication to Vault, and keep the acquired
+token renewed until the renewal is no longer allowed.
+
+![Vault Agent](/assets/images/vault-secure-intro-5.png)
+
+To leverage this feature, run the vault binary in agent mode (`vault agent
+-config=<config_file>`) on the client. The agent configuration file must specify
+the auth method and [sink](/docs/agent/autoauth/sinks/index.html) locations
+where the token to be written.
+
+When the agent is started, it will attempt to acquire a Vault token using the
+auth method specified in the agent configuration file.  On successful
+authentication, the resulting token is written to the sink locations.
+Optionally, this token can be response-wrapped or encrypted. Whenever the
+current token value changes, the agent writes to the sinks. If authentication
+fails, the agent waits for a while and then retry.
+
+The client can simply retrieve the token from the sink and connect to Vault
```

Comment:
I would add: "This simplifies client integration, since the Vault agent handles the login and refresh logic."

Reply:
Thank you!  Incorporated in [147540b](https://github.com/hashicorp/vault/pull/5323/commits/147540b7cda18f2ee14aa4ca3b758e44b6e4d8d8)